nncf/quantization/algorithms/fast_bias_correction/onnx_backend.py

```diff
@@ -95,3 +95,7 @@ def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
     @staticmethod
     def get_node_names_for_input_output_statistics(node: NNCFNode, nncf_graph: NNCFGraph) -> Tuple[str, str]:
         return node.node_name, node.node_name
+
+    @staticmethod
+    def get_activation_channel_axis(node: NNCFNode, port_id: int) -> int:
+        return node.metatype.output_channel_axis
```

Comment:
There are no examples of such models from the ONNX backend. If you have any, share it, please.

Reply:
It looks like you leave a technical debt for you or your colleagues.
From the provided ticket you can try to convert the model into ONNX and try to reproduce. Otherwise, you can create a synthetic model.

Reply:
Do you know any options to convert IR model into ONNX?

Reply:
The original model came from PyTorch, am I right?

Reply:
I see what you mean. Give me some time to completely dive into the way of getting the model and then convert it.
I hope that @alexsu52 and the customers are OK with that postponement.
ONNX and PyTorch model's support with these options is crucial for us, as @kshpv states.

Reply:
Another option could be to separate work into two parts: initially to add support for OpenVINO and then to add the same for ONNX and PyTorch, but the final decision is up to you